packages/agoric-cli/test/upgrade-contract/fixed-contract.js

```diff
@@ -1,7 +1,7 @@
 import { AmountMath } from '@agoric/ertp';
 import { makeDurableZone } from '@agoric/zone/durable.js';
 
-/** @type {ContractMeta} */
+/** @type {ContractMeta<typeof start>} */
```

Comment:
ideally, we'd have a type test showing that this is not a breaking change that forces all clients to change.

one alternative would be to leave the parameter implicit here.

Reply:
agreed. I've reverted a few contracts to not take the generic.